tidb-binlog/tidb-binlog-configuration-file.md

```diff
@@ -140,6 +140,14 @@ summary: TiDB Binlog 配置说明：介绍 Pump 和 Drainer 的配置项，包
 * 对外可访问的 HTTP API 地址，这个地址会被注册到 PD，格式为 `host:port`。
 * 默认：设定成与 `addr` 相同的配置
 
+### load-schema-snapshot 
```

Comment:
in the config file, `load-schema-snapshot` is inside the `[syncer]` section rather than top-level. please put it between the `safe-mode` and `syncer.to` items and use a 4th level header (`#### `).

Reply:
Updated as suggested.